### PR TITLE
Reenable NexusLowRes API 29

### LINKED ISSUE #135784: Firebase API NexusLowRes,version=29, emulator flaky. Re-enable when possible.
More details are available at [ https://console.firebase.google.com/project/flutter-infra-staging/testlab/histories/bh.f93c781079d437/matrices/4629674857943472580 ].
WARNING: Need help for test execution [NexusLowRes-29-en-portrait]? Please join the #test-lab Slack channel at https://firebase.community// and include test matrix ID [matrix-3i93ff86mb1gb] with your question.

| outcome | test_axis_value | test_details |
| --- | --- | --- |
| Passed | Nexus5-21-en-portrait | -- |
| Passed | Nexus5-22-en-portrait | -- |
| Passed | Nexus5-23-en-portrait | -- |
| Passed | Nexus6P-26-en-portrait | -- |
| Passed | Nexus6P-27-en-portrait | -- |
| Inconclusive | NexusLowRes-29-en-portrait | Infrastructure failure |

Current open issue with the FTL team regarding fixing this: https://buganizer.corp.google.com/issues/302753598

### PR DESCRIPTION
**Pull request by ricardoamador**
Reenable the NexusLowRes API version 29 tests. 

*List which issues are fixed by this PR. You must list at least one issue.*
fixes https://buganizer.corp.google.com/issues/302753598
fixes flutter/flutter#135784

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -510,16 +508,15 @@ targets:
...
+      # "--device", "model=Nexus6P,version=25"
       virtual_devices: >-
         [
           "--device", "model=Nexus5,version=21",
           "--device", "model=Nexus5,version=22",
           "--device", "model=Nexus5,version=23",
           "--device", "model=Nexus6P,version=26",
-          "--device", "model=Nexus6P,version=27"
+          "--device", "model=Nexus6P,version=27",
+          "--device", "model=NexusLowRes,version=29"
```
Comment (yusuf-goog) [inline .ci.yaml:519]: nit: add trailing comma everywhere.

**Review by yusuf-goog — APPROVED** [review]
LGTM with nit

File: .ci.yaml (hunk 1 above)
Comment (ricardoamador) [reply .ci.yaml:519]: Is that allowed in yaml?

Comment (ricardoamador) [reply .ci.yaml:519]: Will do.

**Comment by ricardoamador** [comment]
So apparently the .ci.yaml will not parse with the trailing commas for some reason.

**Comment by yusuf-goog** [comment]
> So apparently the .ci.yaml will not parse with the trailing commas for some reason.

Thats weird, looks like trailing commas are in yaml spec?

**Pull request merged**